tests/config/test_configtypes.py

```diff
@@ -70,72 +69,80 @@ def __repr__(self):
                               password=self.password())
 
 
-class ValidValuesTest(unittest.TestCase):
+@pytest.fixture
+def os_path(mocker):
+    """Fixture that mocks and returns os.path from the configtypes module."""
+    return mocker.patch('qutebrowser.config.configtypes.os.path',
+                        autospec=True)
+
+
+class TestValidValues:
 
     """Test ValidValues."""
 
     def test_contains_without_desc(self):
         """Test __contains__ without a description."""
         vv = configtypes.ValidValues('foo', 'bar')
-        self.assertIn('foo', vv)
-        self.assertNotIn("baz", vv)
+        assert 'foo' in vv
+        assert "baz" not in vv
```

Comment:
Mixed usage of single and double quotes. Feel free to ignore this for now as it was like this before.